ckanext/canada/assets/public/canada_public.css

```diff
@@ -576,6 +586,26 @@ ul.resource-list > li.resource-item > div.canada-resource-list-badges{
 }
 /* END - Resource Action Buttons - END */
 
+/* Resource Views List Bar */
+body ul.nav-tabs a[href*="?view_id="]{
```

Comment:
this quite a hack. Is it really that hard to add another icon to the ones in the ckan formats?

Reply:
@wardi yeah, sadly there was no block to easily extend in that template. And there were no specific classes used in any of the html structure, so had to do the dirty CSS selection.

Reply:
A small change in ckan to add a class based on the format would make this a lot cleaner right? Might be worth it because this kind of solution doesn't scale well.

Reply:
Okay I will make an upstream PR with some block or html class

Reply:
@wardi we are actually fully overriding the template in which I would need to add these classes to.

So I have done it in this PR, and will make a ticket to do upstream contrib later.